src/main/java/TechJobs.java

```diff
@@ -1,7 +1,6 @@
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Is there a reason you went with the `*` here?  Just curious. Generally, the * will import all of the classes of the `java.util` package. Which usually means it's loading a lot of unnecessary classes, which causes overhead on the Class itself. just something to keep in mind :)